docs/docs/tools-and-integrations/integrations/k6.md

```diff
@@ -0,0 +1,433 @@
+     http_req_sending...............: avg=47.83µs min=17µs   med=47µs   max=88µs   p(90)=71µs   p(95)=79.49µs 
+     http_req_tls_handshaking.......: avg=0s      min=0s     med=0s     max=0s     p(90)=0s     p(95)=0s      
+     http_req_waiting...............: avg=4.74ms  min=1.75ms med=5.23ms max=7.56ms p(90)=6.69ms p(95)=7.12ms  
+     http_reqs......................: 6       0.350387/s
+     iteration_duration.............: avg=2.44s   min=1s     med=1s     max=11.08s p(90)=5.03s  p(95)=8.06s   
+     iterations.....................: 6       0.350387/s
+     vus............................: 0       min=0      max=1
+     vus_max........................: 1       min=1      max=1
+    [TotalRuns=6, SuccessfulRus=1, FailedRuns=5] 
+[FAILED] 
+[Request=POST - http://localhost:8081/pokemon/import, TraceID=dc071893eaaca9de301f2147e2be372e, RunState=FINISHED FailingSpecs=true, TracetestURL= http://localhost:3000/test/kc_MgKoVR/run/272] 
+[Request=POST - http://localhost:8081/pokemon/import, TraceID=dc0718fff1aca9de30b702c3a1bfad75, RunState=FINISHED FailingSpecs=true, TracetestURL= http://localhost:3000/test/kc_MgKoVR/run/275] 
+[Request=POST - http://localhost:8081/pokemon/import, TraceID=dc0718b8daaca9de301e39889afca15b, RunState=FINISHED FailingSpecs=true, TracetestURL= http://localhost:3000/test/kc_MgKoVR/run/276] 
+[Request=POST - http://localhost:8081/pokemon/import, TraceID=dc0718a7e2aca9de30955b5203b162a7, RunState=FINISHED FailingSpecs=true, TracetestURL= http://localhost:3000/test/kc_MgKoVR/run/273] 
+[Request=POST - http://localhost:8081/pokemon/import, TraceID=dc0718edf9aca9de305916d7b1e7814c, RunState=FINISHED FailingSpecs=true, TracetestURL= http://localhost:3000/test/kc_MgKoVR/run/274] 
+[SUCCESSFUL] 
+[Request=POST - http://localhost:8081/pokemon/import, TraceID=dc0718c9d2aca9de3044a794f7248eab, RunState=FINISHED FailingSpecs=false, TracetestURL= http://localhost:3000/test/kc_MgKoVR/run/271]
+```
+
+Indicating which test failed, the status and links to look at the results directly from the UI.
+
+### Non-Zero status
+
+An important feature is to been able to integrate the extension with your CI/CD tools. In this case the tracetest class allows you to validate the results during the k6 `teardown` lifecycle event which returns a fail status in case any of the test run fails. A failed test can be defined as:
+
+- The trace could not be fetched from the data store
+- Any of the test specs failed
+- Could not connect to the Tracetest API.
+- An execution error from the Tracetest Server side.
+
+To enable this, you can simply add the following to your test script:
+
+```javascript
+// enable this to return a non-zero status code if a tracetest test fails
+export function teardown() {
+  tracetest.validateResult();
+}
+```
+
+### Configuration and Environment variables
+
+If you want to reuse your test scripts for multiple environments and want to avoid hardcoding values like the server url and/or path.
```

Comment:
```suggestion
If you want to reuse your test scripts for multiple environments and to avoid hardcoding values like the server url and/or path, use the following configuration variables to achieve this:
```